pm/sendermonitor.go

```diff
@@ -92,7 +93,8 @@ func NewSenderMonitor(claimant ethcommon.Address, broker Broker, smgr SenderMana
 		smgr:            smgr,
 		tm:              tm,
 		senders:         make(map[ethcommon.Address]*remoteSender),
-		redeemable:      make(chan *SignedTicket),
+		redeemable:      make(chan *redemption),
+		ticketStore:     func() TicketStore { return store },
```

Comment:
If we're adding this state to the struct let's just use the TicketStore directly.

Reply:
fixed in 985649238628b9e8d824d2c54280bd2e84da0466